frontend/cypress/integration/projects/list.spec.ts

```diff
@@ -0,0 +1,36 @@
+///<reference path="../../global.d.ts" />
+import {
+    SEARCH_INPUT,
+    //@ts-ignore
```

Comment:
Out of curiosity: what are we ignoring here and why?

Reply:
Same as other E2E tests. Maybe it's related to previous Node or TS version and can be removed. I'll try